sabycom/src/main/java/ru/tensor/sabycom/widget/SabycomDialog.kt

```diff
@@ -127,18 +127,26 @@ internal class SabycomDialog : BottomSheetDialogFragment() {
      */
     override fun onActivityResult(requestCode: Int, resultCode: Int, intent: Intent?) {
         super.onActivityResult(requestCode, resultCode, intent)
-        webView?.onActivityResult(requestCode, resultCode, intent)
+        binding?.webView?.onActivityResult(requestCode, resultCode, intent)
```

Comment:
@MAKolpakov можем улучшить семантику
```suggestion
binding?.apply {
    webView.onActivityResult(requestCode, resultCode, intent)
}
```